about/about-networking.md

```diff
@@ -0,0 +1,77 @@
+---
+
+You can get up and running with Calico by following any of our getting started guides without needing to be a networking expert. Calico hides the complexities for you.  However, if you would like to learn more about networking so you can better understand what is happening under the covers, this guide provides a short introduction to some of the key fundamental networking concepts for anyone who is not already familiar with them.
+
+In this guide you will learn:
+- The terms used to described different layers of the network.
+- The anatomy of a network packet.
+- What MTU is and why it makes a difference.
+- How IP addressing, subnets, and IP routing works.
+- What an overlay network is.
+- What DNS and NAT are.
+
+### Network layers
+
+The process of sending and receiving data over a network is commonly categorized into 7 layers (referred to as the {% include open-new-window.html text='OSI model' url='https://en.wikipedia.org/wiki/OSI_model' %}). They layers are typically abbreviated as L1 - L7. You can think of data as passing through each of these layers in turn as it is sent or received from an application, which each layer being responsible for a particular part of the processing required to send the data over the network.
+
+![OSI network layers diagram]({{site.baseurl}}/images/osi-network-layers.svg)
+
+In a modern enterprise or public cloud network, the layers commonly map as follows:
+
+- L5-7: all the protocols most application developers are familiar with. e.g. HTTP, FTP, SSH, SSL, DNS.
+- L4: TCP or UDP, including source and destination ports.
+- L3: IP packets and IP routing.
+- L2: Ethernet packets and Ethernet switching.
+
+### Anatomy of a network packet
+
+When sending data over the network, each layer in the network stack adds its own header containing the control/metadata the layer needs to know in order to process the packet as it traverses the network, passing the resulting packet on to the next layer of the stack. In this way the complete packet is produced, which includes all the control/metadata required by every layer of the stack, without any layer understanding the data or needing to process the control/metadata of adjacent network layers. 
+
+![Anatomy of a network packet]({{site.baseurl}}/images/anatomy-of-a-packet.svg)
+
+### MTU
+
+The Maximum Transmission Unit ({% include open-new-window.html text='MTU' url='https://en.wikipedia.org/wiki/Maximum_transmission_unit' %}) of a network is the maximum size of packet that can be sent across the network. When an application tries to send more data than can fit in a single packet, TCP will fragment the packet into multiple TCP segments, so the MTU is not exceeded. Most networks have an MTU of 1,500 bytes, but some networks support MTUs of 9,000 bytes. In a Linux system, larger MTU sizes result in lower CPU being used by the Linux networking stack when sending large amounts of data, because it has to process fewer packets for the same amount of data.
```

Comment:
I think this section is complex (especially if you want to include the extra things I suggest below!), and should be split into a few paragraphs.  Because I've added a forward link, I'd also be tempted to move it to the bottom of the doc?

I'm not sure whether it's helpful to simplify MTU down to being a network level property.  There's configured link MTU, maximum MTUs on all of the switches and hosts that can be reached, MSS and path MTU etc.

Fragmentation is an IP layer process, not TCP layer: it's probably better to use the word segmentation for TCP?  I think it's also worth mentioning that fragmented packets are sometimes dropped by firewalls and switches and it's usually preferable to avoid it.  TCP generally attempts to detect the minimum MTU along the path (path MTU) and sets the maximum segment size (MSS) accordingly to avoid fragmentation.

Performance is more subtle than this largely due to offloading.  I think I'd say something like this for a performance paragraph:
- Larger MTUs reduce the percentage overhead spent transmitting unnecessary headers.  This can be worth up to about 4% moving from 1500 bytes to 9000 bytes.
- Larger MTUs may reduce the CPU required by the Linux networking stack if the kernel has to process fewer packets for the same amount of data.  Many NICs have TCP offloading features that transfer this work to the network card, in which case this benefit does not apply when those features are in use.
- Support for TCP offloading when an (overlay network)[link] is in use is less common and depends on the type of encapsulation, the hardware and the driver configuration.  As such larger MTUs can be have a more significant benefit in these situations.

Reply:
I had a go at making it more accurate and split it into a couple of paragraphs. The risk is it may now be more complicated than is the best fit for this high-level education. I think it's ok, but it feels a bit borderline.